extensions/expath/src/main/java/org/expath/exist/HttpClientModule.java

```diff
@@ -36,7 +36,7 @@
 
     public final static String NAMESPACE_URI = HttpConstants.HTTP_CLIENT_NS_URI;
 
-    public final static String PREFIX = HttpConstants.HTTP_CLIENT_NS_PREFIX;
+    public final static String PREFIX = "http";
```

Comment:
Could you use `HttpConstants.HTTP_NS_PREFIX` instead please?

Reply:
I considered this, but decided against it, since the source code implicitly associates that "prefix" with a different namespace URI. See https://github.com/expath/expath-http-client-java/blob/master/http-client-java/src/main/java/org/expath/httpclient/HttpConstants.java#L19-L20. This namespace URI is not mentioned in the spec. Thus, it seems inappropriate to borrow this value. Effectively, in this PR, we're deciding to hardcode the namespace prefix for eXist, to ensure eXist matches the convention described in the spec, so it's best to prevent future changes to http-client-java package determine what prefix eXist uses.

Reply:
@adamretter Is this a satisfying rationale for the value for `PREFIX`?

Reply:
Sure :-)

Reply:
Great, thanks. Looking forward to 6.0!